2-mandatory/4-restaurant.js

```diff
@@ -20,7 +20,20 @@ const MENU = {
 };
 
 let cashRegister = {
-  // write code here
```

Comment:
This looks good. Again - just keep an eye on the formatting and spacing of your code. For example, your orderBurger function below would be a little easier to read like this:
```javascript
	orderBurger: function (balance) {
		if (balance - MENU.burger >= 0) {
			return balance - MENU.burger;
		} else {
			return balance;
		}
	},
```
One quick way to do this is by using `Format Document` in VsCode.